de-DE/lang/numere.nlng

```diff
@@ -289,6 +289,8 @@ PARSERFUNCS_LISTFUNC_FUNC_STRFND_[STRING]=					strfnd(sToFind,sWhere,nPos=1)
 PARSERFUNCS_LISTFUNC_FUNC_STRING_CAST_[STRING]=				string_cast(EXPR)                                           STR      - Wandelt EXPR ungeachtet m�glicher Rechenoperationen oder Zeichenkettenausdr�cke in eine Zeichenkette um (#(EXPR) w�rde zun�chst den numerischen Wert von EXPR auswerten)
 PARSERFUNCS_LISTFUNC_FUNC_STRIP_[STRING]=					strip(sStr,sPref,sSuff,stripAll=false)                      STR      - Entfernt den Pr�fix sPref und den Suffix sSuff von der Zeichenkette sStr, wenn sie nicht-leer sind. stripAll ist optional und entfernt ebenfalls sich wiederholende Pr�- und Suffixes f�r stripAll = true.
 PARSERFUNCS_LISTFUNC_FUNC_STRLEN_[STRING]=					strlen(sStr)                                                VAL      - Gibt die Anzahl an Zeichen in der Zeichenkette sStr zur�ck
+PARSERFUNCS_LISTFUNC_FUNC_FIRSTCH_[STRING]=					firstch(sStr)                                               VAL      - Gibt das erste Zeichen in der Zeichenkette sStr zur�ck
+PARSERFUNCS_LISTFUNC_FUNC_LASTCH_[STRING]=					lastch(sStr)                                                VAL      - Gibt das letzte Zeichen in der Zeichenkette sStr zur�ck

```

Comment:
Lacks the documentation of `str.first` and `str.last` (see next group of language identifiers)